.ci/Jenkinsfile

```
@@ -247,7 +247,9 @@ def signArtifactsWithElastic() {
     triggerRemoteJob(auth: CredentialsAuth(credentials: 'local-readonly-api-token'),
       job: 'https://internal-ci.elastic.co/job/elastic+unified-release+master+sign-artifacts-with-gpg',
       token: TOKEN,
-      parameters: "gcs_input_path=${env.INFRA_SIGNING_BUCKET_ARTIFACTS_PATH}",
+      parameters: [
+        gcs_input_path: env.INFRA_SIGNING_BUCKET_ARTIFACTS_PATH,
+      ]
```

Missing comma here :)
```suggestion
      ],
```

---

Thanks! I was checking the other block and I didn't see it